integration/api/src/main/resources/ApplicationResources.properties

```diff
@@ -0,0 +1,26 @@
+phenotips.variantDisplay.options.numberOfGenes=Maximum number of genes:
+phenotips.variantDisplay.options.numberOfVariants=Maximum variants per gene:
+phenotips.variantDisplay.optionsButton=Options
+phenotips.variantDisplay.hideOptionsButton=Hide Options
+phenotips.variantDisplay.refreshButton=Refresh
+
+phenotips.variantDisplay.topGenes.header=Top genes in VCF
```

Comment:
Please refer to [the conventions](http://dev.xwiki.org/xwiki/bin/Community/L10N+Conventions#HRulesforstandardformelements), this should end in `.title`, as in `phenotips.variantDisplay.topGenes.title`.

Even better: `phenotips.variants.topGenes.title`. The key names don't have to include the page name in them, that's a bad trend started by Matt (I think), they should instead use a semantic name for the thing where they're used.